### PR TITLE
Routing & Fetching

### PR DESCRIPTION
**Pull request by pupupupupupupupupu**
- Did routing 
- Fetched data from mongoDB

### CODE REVIEW

**Comment by anilgulecha** [comment]
Arsalan, I'm very happy to see the continuous progress here. Is there a URL where I can see the app as well?

**Comment by anilgulecha** [comment]
(my commend wasn't a technical review -- will let a mentor handle that. Just wanted to call out kudos on the progress)

**Comment by pupupupupupupupupu** [comment]
I hosted my website on Netlify last time but I used a build folder for that
so it isn't updated yet. In some days my MVP will be finished so I'll share
the URL then. Hope it's ok.

File: front-end/.env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+REACT_APP_AUTH0_DOMAIN = dev-l5smjk3ej12oyjv8.us.auth0.com
```
Comment (PatelNeelMahesh) [inline front-end/.env:1]: `.env` file should be added in, `.gitignore` & removed from here. It contains sensitive details that should not be available here.

File: front-end/package.json [hunk 2]
```diff
@@ -11,9 +12,16 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.4.0",
     "bootstrap": "^5.3.0-alpha1",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
```
Comment (PatelNeelMahesh) [inline front-end/package.json:18]: You do not need `express` `mongodb` `mongoose` `morgan` in front-end package.json it should be in backend package.json, delete from here.

File: front-end/src/App.js [hunk 3]
```diff
@@ -1,227 +1,33 @@
...
+        <Navbar />
+        <div className="components">
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/comics" element={<Comics />} />
+            <Route path="/upload" element={<Upload />} />
+            <Route path="/aboutus" element={<AboutUs />} />
+          </Routes>
+        </div>
+        {/* <footer>
```
Comment (PatelNeelMahesh) [inline front-end/src/App.js:25]: No need to keep react component <Footer/>  in <footer></footer> tag.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
You have commented code in multiple files, remove it and commit again along with additional changes added here in comments, later if you need commented code you can visit previous commits in github.

**Pull request merged**